operator-scorecard-test.yml

```diff
@@ -43,4 +43,4 @@
             content: "{{ operator_container_result.stdout }}"
             dest: "{{ work_dir }}/scorecard-operator-container-debug.txt"
           when:
-            - operator_container_result.stdout != ""
```

Comment:
This change seems to be related to removing new line from the file.
I know some editors does not add it by default. Or is there any other change?

Reply:
No it looks like that is it. I didn't make this change; it was already in the CVP repository when I copied the files over into this repository. But I can add the newline so that there is no change.